website/docs/r/s3_bucket.html.markdown

```diff
@@ -220,11 +220,17 @@ The `destination` configuration block supports the following arguments:
 * `storage_class` - (Optional) The [storage class](https://docs.aws.amazon.com/AmazonS3/latest/API/API_Destination.html#AmazonS3-Type-Destination-StorageClass) used to store the object. By default, Amazon S3 uses the storage class of the source object to create the object replica.
 * `replica_kms_key_id` - (Optional) Destination KMS encryption key ARN for SSE-KMS replication. Must be used in conjunction with
   `sse_kms_encrypted_objects` source selection criteria.
-* `access_control_translation` - (Optional) Specifies the overrides to use for object owners on replication. Must be used in conjunction with `account_id` owner override configuration.
+* `access_control_translation` - (Optional) Specifies the overrides to use for object owners on replication ([documented below](#access-control-translation)). Must be used in conjunction with `account_id` owner override configuration.
```

Comment:
This, in conjunction with the other suggestion, will make sure that we don't wind up with duplicate anchor links

```suggestion
* `access_control_translation` - (Optional) Specifies the overrides to use for object owners on replication ([documented below](#access_control_translation-block)). Must be used in conjunction with `account_id` owner override configuration.
```